harness/determined/common/storage/__init__.py

```diff
@@ -61,6 +64,13 @@ def build(config: Dict[str, Any], container_path: Optional[str]) -> StorageManag
             config["storage_path"] = config.get("tensorboard_path", None)
         else:
             config["storage_path"] = config.get("checkpoint_path", None)
+    elif identifier == "blob":
```

Comment:
IMO the identifier should be named such that it's clear it's the Azure option. "blob storage" is a pretty generic term for things like S3 and widely misunderstood IMO. Maybe 'azure_storage' or 'azure_blob'?

Reply:
Yup! We've renamed it to azure!